pointerevents/pointerevent_support.js

```diff
@@ -185,3 +193,49 @@ function rPointerCapture(e) {
     catch(e) {
     }
 }
+
+var globalPointerEventTest = null;
+var expectedPointerType = null;
+
+var HOVERABLE_POINTERS = ['mouse', 'pen'];
+
+function MultiPointerTypeTest(testName, types) {
+    this.testName = testName;
+    this.types = types;
+    this.currentTypeIndex = 0;
+    this.createNextTest();
+}
+
+MultiPointerTypeTest.prototype.skip = function() {
+    this.createNextTest().timeout();
```

Comment:
I will rename the function. The order matters because if there is no active test then for example chromium infra passes the test and closes the file. So I need to first create another test to pass the previous one.